packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/tests/index.test.js

```diff
@@ -34,1310 +53,145 @@ describe('Admin | containers | ListView', () => {
-        -webkit-flex-direction: row;
-        -ms-flex-direction: row;
-        flex-direction: row;
-      }
-
-      .c11 {
-        position: relative;
-        outline: none;
-      }
-
-      .c11 > svg {
-        height: 12px;
-        width: 12px;
-      }
-
-      .c11 > svg > g,
-      .c11 > svg path {
-        fill: #ffffff;
-      }
-
-      .c11[aria-disabled='true'] {
-        pointer-events: none;
-      }
+    render(App);
 
-      .c11:after {
-        -webkit-transition-property: all;
-        transition-property: all;
-        -webkit-transition-duration: 0.2s;
-        transition-duration: 0.2s;
-        border-radius: 8px;
-        content: '';
-        position: absolute;
-        top: -4px;
-        bottom: -4px;
-        left: -4px;
-        right: -4px;
-        border: 2px solid transparent;
-      }
+    const loadingElement = await screen.findByText('Loading content.');
 
-      .c11:focus-visible {
-        outline: none;
-      }
+    expect(loadingElement).toBeInTheDocument();
 
-      .c11:focus-visible:after {
-        border-radius: 8px;
-        content: '';
-        position: absolute;
-        top: -5px;
-        bottom: -5px;
-        left: -5px;
-        right: -5px;
-        border: 2px solid #4945ff;
-      }
+    await waitForElementToBeRemoved(() => screen.getByText('Loading content.'));
 
-      .c12 {
-        -webkit-text-decoration: none;
-        text-decoration: none;
-      }
+    expect(screen.getByText('http:://strapi.io')).toBeInTheDocument();
+  });
 
-      .c12[aria-disabled='true'] {
-        border: 1px solid #dcdce4;
-        background: #eaeaef;
-      }
+  it('should show a loader when permissions are loading', () => {
+    useRBAC.mockImplementation(() => ({
+      isLoading: true,
+      allowedActions: { canUpdate: true, canCreate: true, canDelete: true },
+    }));
 
-      .c12[aria-disabled='true'] .c14 {
-        color: #666687;
-      }
+    useNotification.mockImplementation(() => jest.fn());
 
-      .c12[aria-disabled='true'] svg > g,.c12[aria-disabled='true'] svg path {
-        fill: #666687;
-      }
+    const { getByText } = render(App);
 
-      .c12[aria-disabled='true']:active {
-        border: 1px solid #dcdce4;
-        background: #eaeaef;
-      }
+    expect(getByText('Loading content.')).toBeInTheDocument();
+  });
 
-      .c12[aria-disabled='true']:active .c14 {
-        color: #666687;
-      }
+  it('should show a list of webhooks', async () => {
+    useRBAC.mockImplementation(() => ({
+      isLoading: false,
+      allowedActions: { canUpdate: true, canCreate: true, canDelete: true },
+    }));
 
-      .c12[aria-disabled='true']:active svg > g,.c12[aria-disabled='true']:active svg path {
-        fill: #666687;
-      }
+    render(App);
 
-      .c12:hover {
-        border: 1px solid #7b79ff;
-        background: #7b79ff;
-      }
+    await waitFor(() => {
+      expect(screen.getByText('http:://strapi.io')).toBeInTheDocument();
```

Comment:
lets try and avoid screen where we can imo it's more idiomatic to use the returned value from render when using testing-library/react